### PR TITLE
- added wrapper for Md

### PR DESCRIPTION
**Pull request by aschnell**
For https://trello.com/c/JO6m3Nkf/561-3-libstorageng-md-raid-named-raid-array.

### CODE REVIEW

**Comment by aschnell** [comment]
The failures in the tests are due to the tests using an outdated libstorage-ng.

File: test/md_test.rb [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+  describe "#numeric?" do
+
+    it "returns true for /dev/md0" do
+      expect(md.numeric?).to eq true
+    end
+
+  end
+
+  describe "#get_number" do
```
Comment (jreidinger) [inline test/md_test.rb]: this should be `describe "#number" do` as it describe ruby API and not libstorage one.

File: test/md_test.rb [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+    it "returns false since the MD RAID is not in /etc/mdadm.conf" do
+      expect(md.in_etc_mdadm?).to eq false
+    end
+
+  end
+
+  describe "#inspect" do
+
+    it "returns false since the MD RAID is not in /etc/mdadm.conf" do
```
Comment (jreidinger) [inline test/md_test.rb]: copy and paste error.

**Comment by jreidinger** [comment]
in general looks good, just two small notes.

**Comment by aschnell** [comment]
Closed since the coverage calculation is completely broken.

**Comment by aschnell** [comment]
Open new pull request, https://github.com/yast/yast-storage-ng/pull/275.

**Pull request closed without merging**